qa_bands.md

```diff
@@ -20,27 +20,34 @@
 * Does regression and functional testing
 * Writes test documentation (test cases, test scenarios, etc)
 
+### Behaviors
+* Demonstrates proactiveness and communicates testing updates to the team/s
+* Love to learn and develop new skills continuously 
+* Accountable for testing features from start to finish including post production
+
 <a name="qa-engineer"></a>
 ## QA Engineer
 ### Technical Abilities
-* Displays experience working with one or more languages and related ecosystems
 * Possesses strong understanding of QA test process
-* Understands TuneIn's application and codebase
+* Excellent understanding of application that they are testing
+* Ability to run automation in their local environment
+* Write basic  UI automation tests
 
 ### Project Scope
 * Delivers assignments with high quality, readability, and timely delivery
-* Designs solutions to medium complexity testing problems, within constraints appropriate for the platform
 * Coordinates releases to QA and stage environments
-* Coordinates execution plan for distributed QA partners
+* Coordinates and works with offsite QA partners
+* Owns components on their projects or a standalone project
 * Owns small component/area on a few projects or a standalone project
-* You build analytics requirements, dashboards, and alerts to ensure we can observe how our code performs in non-prod and prod environments
+* Start looking at dashboards,alerts to ensure code performs in non-prod and prod environments
+* Maintain release notes 
 
 ### Behaviors
 * You're championing quality throughout your team.
 * Testing is a natural part of your workflow; you feel nervous when we're making changes.
-* You are contributing to best practices/patterns at the software level (testing practices, risk identification and management, coding practices, style guidelines, core libraries, etc.) in the interest of producing quality.
+* You are contributing to best practices/patterns at a quality level (testing practices, risk identification and management etc.) in the interest of producing quality.
```

Comment:
Is this better: You are contributing to best practices/patterns at a software level (testing practices, risk identification and management etc.) by collaborating with developers in the interest of producing quality.